frontend_tests/run-casper

```diff
@@ -72,7 +75,12 @@ def run_tests(files):
             file = os.path.join(os.path.dirname(__file__), '../frontend_tests/casper_tests', file)
         test_files.append(os.path.abspath(file))
 
-    subprocess.check_call('tools/setup/generate-fixtures')
+    if check_if_database_exist('zulip_test_template'):
+        available_migrations = 'var/available-migrations'
+        migration_status = 'var/migration_status'
+        write_migration_status_into_file(available_migrations, settings='zproject.test_settings')
+        if compare_files_if_exist(available_migrations, migration_status):
+            subprocess.check_call('tools/setup/generate-fixtures --force')
```

Comment:
This whole function is the same as the one inside `test-backend`, so I'd move it into the `test_fixtures.py`, but yeah, this is the sort of thing I had in mind!